### PR TITLE
feat: Add iframe support

### PR DESCRIPTION
**Pull request by Matejk00**
Closes: CXSPA-8085

### CODE REVIEW

File: integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html [hunk 1]
```diff
@@ -14,15 +14,23 @@
...
+      }}</a>
+    </div>
+    <ng-container
+      *ngIf="
+        paymentLink.data && paymentLink.renderType === RENDER_PATTERN.IFRAME
+      "
+    >
+      <iframe
+        class="cx-payment-iframe"
         *ngIf="!paymentLink.destination?.form?.length"
```
Comment (FollowTheFlo) [inline integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html]: should we also check url not null (paymentLink.destination?.url) in condition? or iframe tag is smart enough to handle empty value?

File: integration-libs/opf/checkout/styles/components/_opf-checkout-payment-wrapper.scss [hunk 2]
```diff
@@ -11,5 +11,10 @@
     .cx-payment-link {
       text-align: end;
     }
+    .cx-payment-iframe {
+      min-height: 400px;
```
Comment (FollowTheFlo) [inline integration-libs/opf/checkout/styles/components/_opf-checkout-payment-wrapper.scss:15]: just curiosity how was this min-height value 400px decided? i supposed it is standard?

File: integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html (hunk 1 above)
Comment (Matejk00) [reply integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html]: I added additional check, to not render an empty iframe.

File: integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html [hunk 3]
```diff
@@ -14,15 +14,25 @@
...
-        [href]="paymentLink.data"
-        >{{ 'opf.checkout.proceedPayment' | cxTranslate }}</a
-      >
+      <a class="btn btn-primary" [href]="paymentLink.data">{{
+        'opf.checkout.proceedPayment' | cxTranslate
+      }}</a>
+    </div>
+    <ng-container
+      *ngIf="
+        paymentLink.data && paymentLink.renderType === RENDER_PATTERN.IFRAME
```
Comment (FollowTheFlo) [inline integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html:26]: do we also need to cover the use case where renderType is IFRAME and based on script (not url). In my understanding it was the use case we were covering before this fix.

File: integration-libs/opf/checkout/styles/components/_opf-checkout-payment-wrapper.scss (hunk 2 above)
Comment (FollowTheFlo) [reply integration-libs/opf/checkout/styles/components/_opf-checkout-payment-wrapper.scss:15]: as discussed, new jira to handle it as need more analysis and opinions.

**Pull request merged**